skyline-vscode/react-ui/src/App.js

```diff
@@ -139,14 +139,15 @@ function App() {
 
     useEffect(function () {
         window.addEventListener('message', event => {
-            //console.log("Message:", JSON.stringify(event.data));
             if (event.data['message_type'] === "analysis") {
+                console.log("Received data")
```

Comment:
Don't need to log these messages.

Reply:
Deleted console log messages in useEffect call